0032-peering-clearing-settlement/0032-peering-clearing-settlement.md

```diff
@@ -0,0 +1,160 @@
+
+The balance on the Interledger account can only change as a result of two events:
+ 1. The successful routing of an ILP packet between the two peers
+ 1. A payment made between the peers on the underlying payment network they have agreed to use to settle the Interledger account
+
+ This document provides further details on the when these events occur andhow they are handled by the peers. 
+
+> **Example : Alice and Bob**
+
+> - Alice and Bob are peers on the network.  
+> - They agree to exchange ILP packets using the [Bilateral Transfer Protocol (BTP)](../0023-bilateral-transfer-protocol/0023-bilateral-transfer-protocol.md)
+> - They also agree to denominate their Interledger account in Bitcoin (BTC) at a scale of 8 and settle using the Lightning network.
+
+> In this scenario Alice and Bob exchange ILP packets, which they are either sending or forwarding on behalf of another peer, using the BTP protocol. Every ILP Prepare packet has an amount (represented as an unsigned 64-bit integer). Because of how they have agreed to setup their peering relationship a packet with an amount of 5 represents 0.00000005 BTC (5 satoshis).
+
+Once peered, the two connectors both track the Interledger account balance and adjust it for every ILP Packet successfully routed between them. 
+
+When a connector sends an ILP Prepare packet to a peer, and receives a valid ILP Fulfill response, before the expiry of the ILP Prepare, their balance with that peer **decreases**. 
+
+When a connector receives an ILP Prepare packet and returns a valid ILP Fulfill response, before the expiry of the ILP Prepare, their balance with that peer **increases**.
+
+One peer's balance should always be the additive inverse (negation) of the other peer's balance. I.e. If one peer has a balance of 10 the other peer should have a balance of -10.
+
+In financial accounting terms this could be viewed as a revenue/expense account where credits to this account are balanced by a debit to a corresponding "Accounts Receivable" asset account and debits to this account are balanced by a credit to a corresponding "Accounts Payable" liability account.
+
+> **Example : Alice and Bob**
+
+> Assuming they start with a balance of 0 and the following sequence of events occurs:
+
+> 1. Alice passes an ILP Prepare to Bob with an amount of 6.
+> 1. Bob replies, before the request expires, with a valid ILP Fulfill.
+> 1. Bob passes an ILP Prepare to Alice with an amount of 2.
+> 1. Alice is unable to route the packet and replies with an ILP Reject.
+> 1. Bob passes an ILP Prepare to Alice with an amount of 20.
+> 1. Alice replies, before the request expires, with a valid ILP Fulfill.
+
+> The balance changes will be as follows:
+
+> - After Step 2:
```

Comment:
I think it would be clearer to say what the balances are in the list of steps above